runhouse/servers/cluster_servlet.py

```diff
@@ -435,10 +415,32 @@ async def astatus(self):
             "system_disk_usage": disk_usage,
         }
         status_data = ResourceStatusData(**status_data)
+
+        if send_to_den:
+
+            logger.debug("Sending cluster status to Den")
+
+            cluster_uri, api_server_url = get_cluster_properties_for_den_endpoints(
+                status=status_data
+            )
+
+            save_in_den_resp = rns_client.save_status_metrics_to_den(
+                status=status_data,
+                cluster_uri=cluster_uri,
+                api_server_url=api_server_url,
+            )
+
+            response_info = {
```

Comment:
do we need to create new objects and store this? what about logging a warning if it fails, otherwise do nothing?

Reply:
yeah, but since we are sending the status to den from the cluster servlet, the warning log will be logged to the server.log on the cluster. But we want the log to propagate to the user local terminal / notebook / IDE (wherever the user calls `cluster.status()` / `runhouse status`)